.gitignore

```diff
@@ -4,4 +4,4 @@ Cargo.lock
 **/node_modules
 **/.DS_Store
 **/build
-
```

Comment:
I think we should add `**/index.node` too, so that tests/native/index.node doesn't end up in source control.